pkg/controller/elasticsearch/stackmon/beat_config.go

```diff
@@ -36,7 +41,7 @@ func ReconcileConfigSecrets(client k8s.Client, es esv1.Elasticsearch) error {
 		}
 	}
 
-	if monitoring.IsLogsDefined(&es) {
+	if monitoring.IsMetricsDefined(&es) {
```

Comment:
This looks like an unintentional change?

Reply:
@#!, yes.